### DISCUSSION TITLE
Bad URL link in readme of Github Copilot Chat

### DISCUSSION THREAD

**Discussion by keenranger** [community/community#57914] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Link of 'subscription' in [page](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat&ssr=false#overview) is malformed

Current link is
```
https://github.com/community/community/blob/HEAD/%5Burl%5D(https:/github.com/settings/copilot)
```
I think It should be
```
https:/github.com/settings/copilot
```

**Comment by Akash1134** [comment, marked as answer]
Hi @keenranger 

Thank you for highlighting this. I have notified the concerned team to look into it.
